### PR TITLE
Addition of Features/fastlane

### PR DESCRIPTION
**Pull request by AkuaQ**
Addition of fastlane

### CODE REVIEW

**Comment by AkuaQ** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- FirstAppUITests/APIUITests.swift  2
- FirstAppUITests/SignUpUITests.swift  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111&bid=17697648)

File: fastlane/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Install _fastlane_ using
+```
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew cask install fastlane`
+
+# Available Actions
+## iOS
+### ios screenshots
+```
```
Comment (AkuaQ) [inline fastlane/README.md:20]: Codacy Issue found: [[fenced-code-flag] Missing code-language flag](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)

File: fastlane/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Install _fastlane_ using
+```
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew cask install fastlane`
+
+# Available Actions
+## iOS
```
Comment (AkuaQ) [inline fastlane/README.md:18]: Codacy Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)

File: fastlane/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+xcode-select --install
+```
+
+Install _fastlane_ using
+```
+[sudo] gem install fastlane -NV
+```
+or alternatively using `brew cask install fastlane`
+
+# Available Actions
```
Comment (AkuaQ) [inline fastlane/README.md:17]: Codacy Issue found: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)

Comment (AkuaQ) [inline fastlane/README.md:17]: Codacy Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)

File: fastlane/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+fastlane documentation
+================
+# Installation
```
Comment (AkuaQ) [inline fastlane/README.md:3]: Codacy Issue found: [[no-multiple-toplevel-headings] Don’t use multiple top level headings (1:1)](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)

Comment (AkuaQ) [inline fastlane/README.md:3]: Codacy Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5399111)

**Comment by Tiewhan** [comment]
Isn't it that you have to set Codacy to ignore Fastlane's files?

**Comment by Tiewhan** [comment]
To avoid checks like above

**Comment by AkuaQ** [comment]
At first I did that, but Bitrise had a problem with it so I ignored the files in Codacy instead.

**Comment by Tiewhan** [comment]
Yeah yeah, that's what I meant sorry. Bitrise still needs them but just tell Codacy to not check these files. Which is what you seeminlgy did now?

**Comment by AkuaQ** [comment]
Yes, that's what I did.

**Pull request merged**